src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -277,7 +293,11 @@ private void launchChrome() {
      */
     private void launchFirefox() {
         setupDrivermanager(FirefoxDriverManager.getInstance());
-        driver = new FirefoxDriver();
+        String browserName = BrowserType.FIREFOX;
+        final DesiredCapabilities desiredCapabilities = DesiredCapabilities.firefox();
+        Object firefoxOptions = populateBrowserSettings(desiredCapabilities, browserName);
+        desiredCapabilities.setCapability(FirefoxOptions.FIREFOX_OPTIONS, firefoxOptions);
```

Comment:
setting capabilities with options? there seems to be a naming problem.

Reply:
No , you can add the options to an already existing DesiredCapabilities object like this

`DesiredCapabilities` capabilities = DesiredCapabilities.chrome();

// Add the WebDriver proxy capability.
Proxy proxy = new Proxy();
proxy.setHttpProxy("myhttpproxy:3337");
capabilities.setCapability("proxy", proxy);

// Add ChromeDriver-specific capabilities through ChromeOptions.
ChromeOptions options = new ChromeOptions();
options.addExtensions(new File("/path/to/extension.crx"));
capabilities.setCapability(ChromeOptions.CAPABILITY, options);
ChromeDriver driver = new ChromeDriver(capabilities);` 
[Example under Using the ChromeOptions class](https://sites.google.com/a/chromium.org/chromedriver/capabilities)